lib/fast_gettext/translation_repository/merge.rb

```diff
@@ -0,0 +1,71 @@
+require 'fast_gettext/translation_repository/po'
+
+module FastGettext
+  module TranslationRepository
+    # Responsibility:
```

Comment:
what is the difference between this and a Chain ? ... can we just improve chain instead ?